src/halomod/halo_exclusion.py

```diff
@@ -204,17 +230,27 @@ def mask(self):
 
     @property
     def mlim(self):
+        """The mass threshold for the mask."""
         return 4 * np.pi * (self.r / 2) ** 3 * self.mean_density * self.delta_halo / 3
 
     def integrate(self):
+        """
+        This should pass back whatever is multiplied by P_m(k) to get the two-halo
```

Comment:
I think this docstring needs to be more like "Integrate the :meth:`raw_integrand` over mass`" (the original docstring could be moved to the longer description section after a linebreak).